root/static/scripts/common/components/ReleaseEvents.js

```diff
@@ -17,7 +17,9 @@ import isDateEmpty from '../utility/isDateEmpty';
 
 import EntityLink from './EntityLink';
 
-const COLLAPSE_THRESHOLD = 4;
+const TO_TRIGGER_COLLAPSE = 4;
```

Comment:
But if you have `TO_SHOW_BEFORE` and `TO_SHOW_AFTER`, I'm not sure this makes sense to configure separately - I'd think you should just do `const TO_TRIGGER_COLLAPSE = TO_SHOW_BEFORE + TO_SHOW_AFTER + 1` below (or + 2 rather, since you want to hide at least two, and since that's technically the "trigger," but you'll need to change `>` to `>=` below too in that case).